src/app/reconstruction/MACE/PhaseI/ReconSciFi/ReconSciFi.c++

```diff
@@ -115,60 +94,42 @@ auto ReconSciFi::Main(int argc, char* argv[]) const -> int {
                     if (*Get<"t">(*siPMHitRange[j]) >= initialTime and *Get<"t">(*siPMHitRange[j]) < endTime) {
                         initialTime = *Get<"t">(*siPMHitRange[j]);
                         count++;
-                        if (count == sciFiTracker.Threshold()) {
+                        if (count == sciFiTracker.SiPMOpticalPhotonCountThreshold()) {
                             endTime = initialTime + sciFiTracker.TimeWindow();
-
-                            siPMHitData.emplace_back(std::make_shared<Mustard::Data::Tuple<MACE::PhaseI::Data::SiPMHit>>());
+                            siPMHitData.emplace_back(std::make_shared<Mustard::Data::Tuple<MACE::PhaseI::Data::SciFiSimHit>>());
                             *Get<"t">(*siPMHitData.back()) = *Get<"t">(*siPMHitRange[j]);
                             *Get<"EvtID">(*siPMHitData.back()) = *Get<"EvtID">(*siPMHitRange[j]);
                             *Get<"SiPMID">(*siPMHitData.back()) = *Get<"SiPMID">(*siPMHitRange[j]);
```

Comment:
Try to write more compact code.
```suggestion
                            const auto scifiHit{std::make_shared<Mustard::Data::Tuple<MACE::PhaseI::Data::SciFiSimHit>>()};
                            *Get<"t">(*scifiHit) = *Get<"t">(*siPMHitRange[j]);
                            *Get<"EvtID">(*scifiHit) = *Get<"EvtID">(*siPMHitRange[j]);
                            *Get<"FiberID">(*scifiHit) = *Get<"SiPMID">(*siPMHitRange[j]);
                            scifiHitData.emplace_back(std::move(scifiHit));
```
Also this modification is based on the assumption that data model `SciFiHit` should be updated.